modules/editorial-metadata/editorial-metadata.php

```diff
@@ -1,20 +1,21 @@
 <?php
 
 /**
- * Class Editorial_Metadata
+ * Class EditorialMetadata
```

Comment:
We're using both `Class_Name` and `ClassName` in our code. I like the latter, so slowly converting files to use that version.